Documentation/Books/Manual/Views/ArangoSearch/DetailedOverview.md

```diff
@@ -139,13 +165,45 @@ of removing unused segments after release of internal resources.
     upon several possible configurable formulas as defined by their types.
     The currently supported types are:
 
-    - **bytes_accum**: Consolidate if and only if  ({threshold} range `[0.0, 1.0]`)
-      `{threshold} > (segment_bytes + sum_of_merge_candidate_segment_bytes) / all_segment_bytes`,
-      i.e. the sum of all candidate segment's byte size is less than the total
-      segment byte size multiplied by the `{threshold}`.
+    - **bytes_accum**: Consolidate is performed based on current memory cunsumption
+      of segments and `threshold` property value.
     - **tier**: Consolidate based on segment byte size and live document count
       as dictated by the customization attributes.
 
+### `consolidationPolicy` properties for `bytes_accum` type
+  - **threshold** (_optional_; type: `float`; default: `0.1`)
+
+    Defines threshold value of `[0.0, 1.0]` possible range. Consolidation is
+    performed when then the sum of all candidate segment's byte
```

Comment:
Is "candidate segments" actually plural here? Then it should be `candidate segments'`

Reply:
Fixed.